### PR TITLE
python3Packages.fritzconnection: 1.2.1 -> 1.4.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/fritzconnection/default.nix [hunk 1]
```diff
@@ -1,33 +1,22 @@
-{ lib, stdenv, buildPythonPackage, pythonOlder, fetchFromGitHub, pytest, requests }:
+{ lib, stdenv, buildPythonPackage, pythonOlder, fetchFromGitHub, pytestCheckHook, requests }:
 
 buildPythonPackage rec {
   pname = "fritzconnection";
-  version = "1.2.1";
+  version = "1.4.0";
 
+  # no tests on PyPI
```
Comment (Valodim) [inline pkgs/development/python-modules/fritzconnection/default.nix:7]: :+1:

File: pkgs/development/python-modules/fritzconnection/default.nix [hunk 2]
```diff
@@ -1,33 +1,22 @@
...
+  disabled = pythonOlder "3.6";
 
   propagatedBuildInputs = [ requests ];
 
-  checkInputs = [ pytest ];
-
-  checkPhase = ''
-    pytest
-  '';
+  checkInputs = [ pytestCheckHook ];
```
Comment (Valodim) [inline pkgs/development/python-modules/fritzconnection/default.nix:19]: :+1:

**Review by Valodim — APPROVED** [review]
lgtm

**Review by mweinelt — CHANGES_REQUESTED** [review]
The new homepage is https://github.com/kbr/fritzconnection.

**Review by mweinelt — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 109525` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.fritzconnection</li>
    <li>python38Packages.fritzconnection</li>
    <li>python39Packages.fritzconnection</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109525` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.fritzconnection</li>
<li>python38Packages.fritzconnection</li>
<li>python39Packages.fritzconnection</li>
</ul>
</details>

**Pull request merged**